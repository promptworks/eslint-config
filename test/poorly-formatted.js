import React from 'react';
import foo from 'foo';

var a = 1;
let b = 2;

console.log(a, b);

const MyComponent = () => {
  <div>
    <label>Hi I'm Ray!</label>
  </div>
};
