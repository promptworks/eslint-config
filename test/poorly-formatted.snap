{"results":[{"filePath":"/Users/rzane/Workspace/eslint-config-promptworks/test/poorly-formatted.js","messages":[{"ruleId":"no-unused-vars","severity":2,"message":"'foo' is defined but never used. Allowed unused vars must match /^_/.","line":2,"column":8,"nodeType":"Identifier","source":"import foo from 'foo';","endLine":2,"endColumn":11},{"ruleId":"import/no-unresolved","severity":2,"message":"Unable to resolve path to module 'foo'.","line":2,"column":17,"nodeType":"Literal","source":"import foo from 'foo';","endLine":2,"endColumn":22},{"ruleId":"no-var","severity":2,"message":"Unexpected var, use let or const instead.","line":4,"column":1,"nodeType":"VariableDeclaration","source":"var a = 1;","endLine":4,"endColumn":11,"fix":{"range":[51,54],"text":"let"}},{"ruleId":"prefer-const","severity":2,"message":"'b' is never reassigned. Use 'const' instead.","line":5,"column":5,"nodeType":"Identifier","source":"let b = 2;","endLine":5,"endColumn":6,"fix":{"range":[62,65],"text":"const"}},{"ruleId":"no-console","severity":2,"message":"Unexpected console statement.","line":7,"column":1,"nodeType":"MemberExpression","source":"console.log(a, b);","endLine":7,"endColumn":12},{"ruleId":"no-undef","severity":2,"message":"'console' is not defined.","line":7,"column":1,"nodeType":"Identifier","source":"console.log(a, b);","endLine":7,"endColumn":8},{"ruleId":"no-unused-vars","severity":2,"message":"'MyComponent' is assigned a value but never used. Allowed unused vars must match /^_/.","line":9,"column":7,"nodeType":"Identifier","source":"const MyComponent = () => {","endLine":9,"endColumn":18},{"ruleId":"jsx-a11y/label-has-for","severity":2,"message":"Form label must have associated control","line":11,"column":5,"nodeType":"JSXOpeningElement","source":"    <label>Hi I'm Ray!</label>","endLine":11,"endColumn":12},{"ruleId":"react/no-unescaped-entities","severity":2,"message":"HTML entities must be escaped.","line":11,"column":16,"nodeType":"Literal","source":"    <label>Hi I'm Ray!</label>"},{"ruleId":"prettier/prettier","severity":2,"message":"Insert `;`","line":12,"column":9,"nodeType":null,"source":"  </div>","endLine":12,"endColumn":9,"fix":{"range":[169,169],"text":";"}}],"errorCount":10,"warningCount":0,"fixableErrorCount":3,"fixableWarningCount":0,"source":"import React from 'react';\nimport foo from 'foo';\n\nvar a = 1;\nlet b = 2;\n\nconsole.log(a, b);\n\nconst MyComponent = () => {\n  <div>\n    <label>Hi I'm Ray!</label>\n  </div>\n};\n"}],"errorCount":10,"warningCount":0,"fixableErrorCount":3,"fixableWarningCount":0}